<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/icon/icon/kdu-beauty.ico</normaloff>:/icon/icon/kdu-beauty.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
* General
*/
QToolButton {
background-color:transparent;
border-style:none;
}

QTabWidget, QListView, QTextEdit, QTableWidget {
border-style:none;
}

/*
* Main Region
*/
QWidget#widgetTombol {
/*Default*/
/* background-color:rgb(7, 51, 97); */
/*background-color: rgb(137, 51, 186);*/
background-color:rgb(10, 115, 255)
}

QWidget#widgetTombol QLabel#labelJudul {
color:rgb(85, 255, 255);
font-weight:bold;
font-size:24px;
}

QWidget#widgetTombol QToolButton {
min-width:80px;
min-height:30px;
padding: 12px 8px;
border-radius:8px;
color:rgb(255, 255, 255);
}

QWidget#widgetTombol QToolButton:hover {
background-color:rgb(0, 170, 0);
color:rgb(0, 0, 0);
}

QLabel#labelVersi {
color: rgb(255, 255, 255);
}



/*
* Editor Region
*/
QWidget#widgetTitleBar QToolButton {
padding: 4px 12px;
}

QToolButton#tbExit:hover {
background-color:rgb(170, 85, 0);
}

QToolButton#tbMaximize:hover,
QToolButton#tbMinimize:hover {
background-color:rgb(170, 170, 0);
}

QSplitter#splitter {
margin:0 16px;
}

QTabWidget::pane {
border-top:2px solid rgb(0, 170, 255);
}

QTabWidget QTabBar::tab {
padding: 4px 12px;
border-top-right-radius:4px;
border-top-left-radius:4px;
}

QTabWidget QTabBar::tab:selected {
background-color:rgb(0, 170, 255);
}

QTabWidget QTabBar::tab:!selected {
background-color:rgb(0, 255, 0);
margin-top:4px;
}

QTableWidget &gt; QWidget {
background-color:rgb(255, 255, 255);
}

/* Widget Shortcut */
QWidget#widgetShortcut QLabel {
color: rgb(255, 255, 255);
}

QPushButton#tombolKembali {
color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="centralLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widgetTombol" native="true">
      <property name="minimumSize">
       <size>
        <width>225</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="layoutMain">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>32</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QLabel" name="labelJudul">
         <property name="text">
          <string notr="true">Data Extractor</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutTombol">
         <property name="spacing">
          <number>24</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QToolButton" name="tombolFolder">
           <property name="text">
            <string notr="true">Pilih Folder</string>
           </property>
           <property name="icon">
            <iconset resource="assets.qrc">
             <normaloff>:/icon/icon/folder.svg</normaloff>:/icon/icon/folder.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QToolButton" name="tombolShortcut">
           <property name="text">
            <string notr="true">Tampilkan
Shortcut</string>
           </property>
           <property name="icon">
            <iconset resource="assets.qrc">
             <normaloff>:/icon/icon/keyboard.svg</normaloff>:/icon/icon/keyboard.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widgetShortcut" native="true">
           <layout class="QGridLayout" name="layoutShortcut">
            <property name="horizontalSpacing">
             <number>24</number>
            </property>
            <item row="7" column="0">
             <widget class="QLabel" name="labelC3a">
              <property name="text">
               <string notr="true">Copy ke
Kolom 3</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QPushButton" name="tombolKembali">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string notr="true">Kembali</string>
              </property>
              <property name="text">
               <string notr="true">Kembali</string>
              </property>
              <property name="icon">
               <iconset resource="assets.qrc">
                <normaloff>:/icon/icon/undo.svg</normaloff>:/icon/icon/undo.svg</iconset>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="labelC1a">
              <property name="text">
               <string notr="true">Copy ke
Kolom 1</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="labelMTa">
              <property name="text">
               <string notr="true">Pindahkan Resi
ke Tab Selesai</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelMNa">
              <property name="text">
               <string notr="true">Pindah ke
Baris Sebelumnya</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="labelMNb">
              <property name="text">
               <string notr="true">W</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelMPa">
              <property name="text">
               <string notr="true">Pindah ke
Baris Selanjutnya</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="labelC3b">
              <property name="text">
               <string notr="true">3</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLabel" name="labelMTb">
              <property name="text">
               <string notr="true">Space</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="labelARb">
              <property name="text">
               <string notr="true">A</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="labelMVb">
              <property name="text">
               <string notr="true">S</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelARa">
              <property name="text">
               <string notr="true">Tambah Baris</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="labelC2a">
              <property name="text">
               <string notr="true">Copy ke
Kolom 2</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="labelC2b">
              <property name="text">
               <string notr="true">2</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="labelC1b">
              <property name="text">
               <string notr="true">1</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="labelDRb">
              <property name="text">
               <string notr="true">D</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelDRa">
              <property name="text">
               <string notr="true">Hapus Baris</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutAbout">
         <item alignment="Qt::AlignBottom">
          <widget class="QLabel" name="labelVersi">
           <property name="text">
            <string notr="true">v0.1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignBottom">
          <widget class="QLabel" name="labelLink">
           <property name="toolTip">
            <string notr="true">Lihat Update</string>
           </property>
           <property name="text">
            <string notr="true">&lt;html&gt;&lt;head&gt;&lt;style&gt;a {text-decoration:none;color:rgb(128,200,128);}&lt;/style&gt;&lt;/head&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://www.github.com/obi1529&quot;&gt;GitHub&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetEditor" native="true">
      <layout class="QVBoxLayout" name="layoutEditor" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetTitleBar" native="true">
         <layout class="QHBoxLayout" name="layoutTitleBar">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>48</number>
          </property>
          <item>
           <spacer name="titleBarHorizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="tbMinimize">
            <property name="toolTip">
             <string notr="true">Minimize</string>
            </property>
            <property name="text">
             <string notr="true">Min</string>
            </property>
            <property name="icon">
             <iconset resource="assets.qrc">
              <normaloff>:/icon/icon/window-minimize.svg</normaloff>:/icon/icon/window-minimize.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbMaximize">
            <property name="toolTip">
             <string notr="true">Maximize/Restore</string>
            </property>
            <property name="text">
             <string notr="true">Max</string>
            </property>
            <property name="icon">
             <iconset resource="assets.qrc">
              <normaloff>:/icon/icon/window-restore.svg</normaloff>:/icon/icon/window-restore.svg</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbExit">
            <property name="toolTip">
             <string notr="true">Tutup Aplikasi</string>
            </property>
            <property name="text">
             <string notr="true">Ext</string>
            </property>
            <property name="icon">
             <iconset resource="assets.qrc">
              <normaloff>:/icon/icon/x.svg</normaloff>:/icon/icon/x.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="handleWidth">
          <number>8</number>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QSplitter" name="widgetPdf">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>8</number>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tabProses">
            <attribute name="title">
             <string notr="true">Proses</string>
            </attribute>
            <layout class="QHBoxLayout" name="layoutTabProses">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="listWidgetProses"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tabSelesai">
            <attribute name="title">
             <string notr="true">Selsai</string>
            </attribute>
            <layout class="QHBoxLayout" name="LayoutTabSelesai">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="listWidgetSelesai"/>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QTextEdit" name="textEdit">
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="placeholderText">
            <string notr="true"/>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widgetExcel" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="layoutExcel">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="toolLayout">
             <item>
              <widget class="QToolButton" name="tbTambahRow">
               <property name="toolTip">
                <string notr="true">Tambah Row</string>
               </property>
               <property name="text">
                <string notr="true">Add</string>
               </property>
               <property name="icon">
                <iconset resource="assets.qrc">
                 <normaloff>:/icon/icon/plus.svg</normaloff>:/icon/icon/plus.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbHapusRow">
               <property name="toolTip">
                <string notr="true">Hapus Row</string>
               </property>
               <property name="text">
                <string notr="true">Rmv</string>
               </property>
               <property name="icon">
                <iconset resource="assets.qrc">
                 <normaloff>:/icon/icon/minus.svg</normaloff>:/icon/icon/minus.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbOpen">
               <property name="toolTip">
                <string notr="true">Muat File Excel</string>
               </property>
               <property name="text">
                <string notr="true">Opn</string>
               </property>
               <property name="icon">
                <iconset resource="assets.qrc">
                 <normaloff>:/icon/icon/excel.svg</normaloff>:/icon/icon/excel.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbSave">
               <property name="toolTip">
                <string notr="true">Simpan</string>
               </property>
               <property name="text">
                <string notr="true">Sav</string>
               </property>
               <property name="icon">
                <iconset resource="assets.qrc">
                 <normaloff>:/icon/icon/save.svg</normaloff>:/icon/icon/save.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="toolHorizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelStartCell">
               <property name="text">
                <string notr="true">Start Cell :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leCell">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>82</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string notr="true">Mulai dari Cell...</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="maxLength">
                <number>10</number>
               </property>
               <property name="frame">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string notr="true">Mulai dari Cell</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelFileName">
               <property name="text">
                <string notr="true">File Name :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leFileName">
               <property name="toolTip">
                <string notr="true">Nama File</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="frame">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string notr="true">Nama File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbClear">
               <property name="toolTip">
                <string notr="true">Bersihkan Tabel</string>
               </property>
               <property name="text">
                <string notr="true">Rf</string>
               </property>
               <property name="icon">
                <iconset resource="assets.qrc">
                 <normaloff>:/icon/icon/refresh.svg</normaloff>:/icon/icon/refresh.svg</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="rowCount">
              <number>1</number>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <row/>
             <column>
              <property name="text">
               <string notr="true">Nama Customer</string>
              </property>
              <property name="toolTip">
               <string notr="true"/>
              </property>
              <property name="whatsThis">
               <string notr="true"/>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">Nomor Telepon</string>
              </property>
              <property name="toolTip">
               <string notr="true"/>
              </property>
              <property name="whatsThis">
               <string notr="true"/>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">Alamat</string>
              </property>
              <property name="toolTip">
               <string notr="true"/>
              </property>
              <property name="whatsThis">
               <string notr="true"/>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="toolTip">
               <string notr="true"/>
              </property>
              <property name="whatsThis">
               <string notr="true"/>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
